packages/app/components/Today/ReleaseNotes.tsx

```diff
@@ -0,0 +1,71 @@
+import { Modal, notification } from "antd";
+import { ReactElement, useState, useEffect } from "react";
+import { useLocalStorage } from "../../hooks/useLocalStorage";
+import { NotionRenderer } from "react-notion";
+
+export default function ReleaseNotes(): ReactElement {
+  const [releaseNotesLastSeen, setReleaseNotesLastSeen] = useLocalStorage(
+    "releaseNotesLastSeen",
+    null
+  );
+  const [releaseNotesLastUpdated, setReleaseNotesLastUpdated] = useState(0);
+  const [notionReleaseNotes, setNotionReleaseNotes] = useState();
+  const [showReleaseNotes, setShowReleaseNotes] = useState(false);
+
+  if (!releaseNotesLastSeen) {
+    setReleaseNotesLastSeen(new Date());
+  }
+
+  useEffect(() => {
+    fetch(
+      "https://notion-api.splitbee.io/v1/page/abba246bfa0847baa2706ab30d0c6c7d"
+    )
+      .then((res) => res.json())
+      .then(
+        (result) => {
+          const timeText =
+            result["beae2a02-249e-4b61-9bfc-81258d93f20d"]?.value?.properties
+              ?.title[0][0];
+          setReleaseNotesLastUpdated(timeText.split("Unix ")[1] * 1000);
+          // Remove the time block and page link block from page
+          result["beae2a02-249e-4b61-9bfc-81258d93f20d"].value.properties.title = []
+          result["4d25f393-e570-4cd5-ad66-b278a0924225"].value.properties.title = []
+          setNotionReleaseNotes(result);
+        },
+        (error) => {
+          console.log("Error fetching release notes", error);
+        }
+      );
+  }, []);
+
+  if (new Date(releaseNotesLastSeen) < new Date(releaseNotesLastUpdated)) {
+    notification.open({
```

Comment:
It'd be nice if this notification had styling `cursor: pointer` so that when you mouse over it, it looks clickable

Reply:
good call, just added.